resources/svc/default-postgresql.yml

```diff
@@ -1,17 +1,18 @@
-version: '3.1'
+# This configuration is intended for development purpose, it's **your** responsibility to harden it for production
+version: '3.7'
 
 services:
   postgresql:
-    image: postgres
-    restart: always
+    image: postgres:13.4
+    # If you want to relocate the postgres data (volume) to your local directory, uncomment the lines below
+    # volumes:
+    #   - ~/volumes/entando-bundle-cli/%BUNDLENAME%/postgresql/:/var/lib/postgresql/data/
     environment:
-      - POSTGRES_USER=%BUNDLENAME%-user
-      - POSTGRES_PASSWORD=adminadmin
-      - POSTGRES_DB=%BUNDLENAME%-db
+      - POSTGRES_USER=%BUNDLENAME%
+      - POSTGRES_PASSWORD=
```

Comment:
empty password?

Reply:
I added username and password as you suggested.